lib/elixir/lib/string.ex

```diff
@@ -591,19 +591,26 @@ defmodule String do
   defdelegate rstrip(binary), to: String.Break
 
   @doc """
-  Returns a string where all trailing `char`s have been removed.
+  Returns a string where all trailing `to_strip`s have been removed.
 
   ## Examples
 
       iex> String.rstrip("   abc _", ?_)
       "   abc "
 
+      iex> String.rstrip("ab abc ab", "ab")
+      "ab abc "
+
   """
-  @spec rstrip(t, char) :: t
+  @spec rstrip(t, char | binary) :: t
   def rstrip(string, char) when is_integer(char) do
```

Comment:
Since we have two clauses, it would probably be better to have a bodyless so that the argument names are taken from that:

```elixir
@spec ...
def rstrip(string, to_strip)

def rstrip(string, char) when is_integer(char) do
...
```

The way it is now, the documentation will show the function as `rstrip(string, char)` but you mention `to_strip` in the documentation.